src/components/throughput-widget/throughput-widget.tsx

```diff
@@ -40,14 +40,11 @@ export class ThroughputWidget {
 
     this.checkAuth();
 
-    if (window.location.hash) {
```

Comment:
As isamples uses a hash router, this would always be true. Instead, check if we need to initialize a throughput token and if access token is added as a query param.